doc/src/cli.rst

```diff
@@ -0,0 +1,81 @@
+.. highlight:: none
+
+**********
+Command Line Interface
+**********
+
+.. _cli:
+
+More information on the commands and there inputs can be found by using 
+``--help``. Example::
+
+     pyfr run --help
+
+Major Commands
+==============
+
+There are four major commands of PyFR, these are:
+
+1. ``pyfr import`` --- convert a `Gmsh
+   <http:http://geuz.org/gmsh/>`_ .msh file into a PyFR .pyfrm file.
+
+   Example::
+
+        pyfr import mesh.msh mesh.pyfrm
+
+2. ``pyfr partition`` --- partition an existing mesh and
+   associated solution files.
+
+   Example::
+
+        pyfr partition 2 mesh.pyfrm solution.pyfrs .
+
+3. ``pyfr run`` --- start a new PyFR simulation. Example::
+
+        pyfr run mesh.pyfrm configuration.ini
+
+4. ``pyfr restart`` --- restart a PyFR simulation from an existing
+   solution file. Example::
+
+        pyfr restart mesh.pyfrm solution.pyfrs
+
+
+Plugin Commands
+===============
+
+Plugins are able to define tools that are accessible from the command line, 
+below are listed the the various options currently available. 
+
+Time Average
+^^^^^^^^^^^^
+
+1. ``pyfr tavg merge`` --- merge together multiple windowed time average files 
+   into a single time average file. The time windows are read from the files 
+   and do no need to be even spaced in time.
+   Example::
+
+        pyfr tavg merge avg-1.00.pyfrs avg-2.00.pyfrs merged_avg.pyfrs
+
+
+Writer
```

Comment:
Are you okay to shelve this for the moment? It is a bit confusing to use the writer plugin to export time average files.

Reply:
Yep, sorry I forgot to update the documentation